### PR TITLE
cdc,pkg,tests(ticdc): unify cdc log style

### LINKED ISSUE #4239: Unify log style and zap field names
* Prefer using "duration" for slow logs
* Prefer using camelCase for zap field names

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

* Prefer using "duration" for slow logs
* Prefer using camelCase for zap field names

Issue Number:  close #4239

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: scripts/check-log-style.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# zap field name should be camelCase, excepts for idioms and special terms.
+grep -RnE "zap.[A-Z][a-z]+\(\".*-.*\"" cdc tests pkg |
```
Comment (amyangfei) [inline scripts/check-log-style.sh:15]: `zap.String("node name", r.name)`
This zap filed style (words split by space) is not checked.

Comment (overvenus) [reply scripts/check-log-style.sh]: Addressed, thanks!

**Review by amyangfei — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

**Pull request merged**